Management_System/BLL/Services/Implementaiton/BoardService.cs

```diff
@@ -0,0 +1,45 @@
+﻿using BLL.Services.Interfaсes;
+using DAL.Entities;
+using DAL.Repositories.Interfaces;
+
+namespace BLL.Services.Implementaiton;
+public class BoardService(IBoardRepository repository) : IBoardService
+{
+    public async Task<BoardEntity> Add(BoardEntity entity, CancellationToken cancellationToken)
+    {
+        await repository.Add(entity, cancellationToken);
+
+        return entity;
```

Comment:
Is the instance of entity will be updated?

Reply:
I'll change entity to dto later (if I get you right)

Reply:
I mean that at the beginning of the method you have entity without id. This field will be set up by DB. So, don't you need to have full entity after the query has finished?